vita3k/gui/src/settings_dialog.cpp

```diff
@@ -416,10 +419,20 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
     } else
         ImGui::PopStyleColor();
 
-    if (host.cfg.overwrite_config)
-        config::serialize_config(host.cfg, host.cfg.config_path);
-
     ImGui::EndTabBar();
+
+    if (host.cfg.overwrite_config) {
+        ImGui::Spacing();
+        ImGui::Separator();
+        ImGui::Spacing();
+        static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+        ImGui::SetCursorPosX((ImGui::GetWindowSize().x / 2.f) - (BUTTON_SIZE.x / 2.f));
+        if (ImGui::Button("Save", BUTTON_SIZE))
+            config::serialize_config(host.cfg, host.cfg.config_path);
+        if (ImGui::IsItemHovered())
+            ImGui::SetTooltip("Click on Save is required to keep changes after reboot.");
```

Comment:
```suggestion
            ImGui::SetTooltip("Click Save to keep changes after reboot.");
```

Reply:
Hmm bent have already review and fix it 😅

Reply:
Alright, I'll do a separate PR for all the grammar stuff lol.